File: Backend/src/main/java/com/bsep/tim11/bseptim11/model/UserTokenState.java
package com.bsep.tim11.bseptim11.model;

public class UserTokenState {

    private String accessToken;
    private Long expiresIn;

    public UserTokenState() {
        this.accessToken = null;
        this.expiresIn = null;
    }

    public UserTokenState(String accessToken, long expiresIn) {
        this.accessToken = accessToken;
        this.expiresIn = expiresIn;
    }

    public void setAccessToken(String accessToken) {
        this.accessToken = accessToken;
    }

    public String getAccessToken() {
        return accessToken;
    }

    public void setExpiresIn(Long expiresIn) {
        this.expiresIn = expiresIn;
    }

    public Long getExpiresIn() {
        return expiresIn;
    }

}

File: Frontend/src/app/models/userTokenState.ts
export class UserTokenState {

  public jwtAccessToken: string;
  public expiresIn: number;

	constructor($jwtAccessToken: string, $expiresIn: number) {
		this.jwtAccessToken = $jwtAccessToken;
		this.expiresIn = $expiresIn;
	}

}